src/components/UserDashboard/Ontology/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import CardHeader from 'uikit/Card/CardHeader';
+import ChartContentSpinner from 'components/Charts/ChartContentSpinner';
+
+import { PhenotypeStore } from '../../OntologyBrowser/store';
+
+import { DashboardCard } from '../styles';
+import './Ontology.css';
+import Sunburst from '../../Charts/Sunburst';
+
+type OntologySunburstProps = {};
+type OntologySunburstState = {
+  data: Object | null;
+};
+
+type FormatterDataType = { title: string; results: number };
+
+const tooltipFormatter = (data: FormatterDataType) => `${data.results}\n\n${data.title}`;
+const centerTextFormatter = (data: FormatterDataType) => `${data.results} ${data.title}`;
+
+class OntologySunburst extends React.Component<OntologySunburstProps, OntologySunburstState> {
+  state = {
+    data: null,
+  };
+
+  componentDidMount() {
+    const ontologyStore = new PhenotypeStore();
+    ontologyStore
+      .fetch()
```

Comment:
I think async/await looks better but it's a personal taste...why do you think of it?